lib/ansible/plugins/connection/ssh.py

```diff
@@ -260,7 +260,7 @@
             version_added: '2.7'
       use_tty:
         version_added: '2.5'
-        default: 'yes'
+        default: 'yes'  forced to no for OpenVMS
```

Comment:
dont add comments to default values, this breaks default processing

Reply:
updated.